controller/setting_controller.go

```diff
@@ -965,6 +970,34 @@ func (sc *SettingController) updateLogLevel() error {
 	return nil
 }
 
+func (sc *SettingController) syncDefaultLonghornStaticStorageClass() error {
+	setting, err := sc.ds.GetSettingWithAutoFillingRO(types.SettingNameDefaultLonghornStaticStorageClass)
+	if err != nil {
+		return err
+	}
+
+	defaultStaticStorageClassName := setting.Value
+	_, err = sc.ds.GetStorageClassRO(defaultStaticStorageClassName)
+	if err != nil && apierrors.IsNotFound(err) {
+		allowVolumeExpansion := true
+		storageClass := &storagev1.StorageClass{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: defaultStaticStorageClassName,
+			},
+			Provisioner:          types.LonghornDriverName,
+			AllowVolumeExpansion: &allowVolumeExpansion,
+			Parameters:           map[string]string{"staleReplicaTimeout": "30"},
```

Comment:
Only 30 seconds?

Reply:
That's 30 minutes. https://github.com/longhorn/longhorn/issues/6446#issuecomment-1861816858